src/mem/ruby/system/Sequencer.cc

```diff
@@ -52,6 +52,7 @@
 #include "debug/RubyHitMiss.hh"
 #include "debug/RubySequencer.hh"
 #include "debug/RubyStats.hh"
+#include "gpu-compute/gpu_dyn_inst.hh"
```

Comment:
This include is going to cause all kinds of problems because the GPU model is not built by default

Reply:
This would definitely need to be wrapped in BUILD_GPU macro if we keep this